deploy/ui/form.json

```diff
@@ -3088,6 +3088,30 @@
                   "description": "The image tag to use for Process Instance Migration e.g. 7.9.0, this param is optional for custom image.",
                   "type": "text"
                 },
+                {
+                  "label": "Process Instance Migration Admin Username, if empty the common AdminUser will be used.",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.processMigration.username",
+                  "type": "text"
+                },
+                {
+                  "label": "Process Instance Migration Admin Password, if empty the common AdminPassword will be used.",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.processMigration.password",
+                  "type": "password"
+                },
+                {
+                  "label": "Process Instance Migration ExtraClassPath Allows to add extra jars to the application classpath separated by colon. Needs to be mounted on the image before.",
```

Comment:
This is not used

Reply:
if user needs somehow pass any extra jar, a custom jdbc extension?
it will work similarly, but instead add classpath, it will be copied to lib/deployment directory to be augmented.

Reply:
https://github.com/jboss-container-images/jboss-kie-modules/pull/489/files#diff-fdad7bb2bd3f151f1e14f026daa5e91c6acceaff166c8ffeabc7baa8ed0abd5aR24

Reply:
ah great, didn't know about this step